src/targets/gpu/hip.cpp

```diff
@@ -108,8 +108,10 @@ static host_ptr_cache& get_host_ptr_cache()
 
 std::shared_ptr<void> allocate_gpu(std::size_t sz, bool host = false)
 {
+    #ifndef _WIN32
     if(sz > get_available_gpu_memory())
         MIGRAPHX_THROW("Memory not available to allocate buffer: " + std::to_string(sz));
```

Comment:
Is `hipMemGetInfo` doesn't work on windows ? 

Also simply disabling the check may not print useful message when it fails.  It would need to rely on hipHostMalloc or hipMalloc to produce error code

Reply:
We assume it's not working correctly . This will be a workaround for our testing. You can find more information at the following links. Comments on this: https://confluence.amd.com/display/VPGFXAT/Stable+Diffusion+malfunction+investigation?focusedCommentId=1372236680&refresh=1720075365089#comment-1372236680 , and here https://ontrack-internal.amd.com/browse/SWDEV-471422

Reply:
We assume it's not working correctly . This will be a workaround for our testing.